setup.cfg

```diff
@@ -43,11 +43,6 @@ filterwarnings =
     ignore:.*in an upcoming release, it will be required to pass the indexing argument.*:UserWarning:torch.*:
```

Comment:
I think we could do a little more work with the `torch` related warnings, but I think we might be able to tackle those as part of https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4345, given that we're upgrading torch in that PR.

Reply:
Updated in https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4355/commits/e8c2cee755f995d1aaf3781329357dfae7610dea.